common/primitives/src/parquet/column.rs

```diff
@@ -24,7 +27,17 @@ impl ParquetColumn {
 		column_type: ParquetType,
 		compression: ColumnCompressionCodec,
 		bloom_filter: bool,
+		optional: bool,
 	) -> ParquetColumn {
-		ParquetColumn { name, column_type, compression, bloom_filter }
+		ParquetColumn {
+			name,
+			column_type,
+			compression,
+			bloom_filter,
+			optional: match optional {
+				true => Some(true),
+				false => None,
+			},
+		}
```

Comment:
I think there is a shorthand 

`optional.then_some(true)` just not sure if it works